### ISSUE TITLE
pretraining masked modeling loss curve

### ISSUE THREAD

**Issue by wuycM** [pluskal-lab/DreaMS#28]
Hi, thanks for your great work on Dreams! 
I'm currently trying to reproduce the pretraining results, and I was wondering if you could share the masked modeling loss curve (or the loss log) from the pretraining stage. It would really help to verify that my training is progressing similarly to yours.
